corehq/messaging/scheduling/forms.py

```diff
@@ -2970,7 +2970,10 @@ def add_additional_content_types(self):
                 (self.CONTENT_CUSTOM_SMS, _("Custom SMS")),
             ]
 
-        if settings.FCM_CREDS or self.initial.get('content') == self.CONTENT_FCM_NOTIFICATION:
```

Comment:
nit: I believe this could be feature flag enabled and (settings OR content) check. Easier to read that way.